README.md

```diff
@@ -110,6 +110,13 @@ sudo git clone https://github.com/ahmetb/kubectx /opt/kubectx
 sudo ln -s /opt/kubectx/kubectx /usr/local/bin/kubectx
 sudo ln -s /opt/kubectx/kubens /usr/local/bin/kubens
 ```
+
+Also available as a Debian package (for [Debian Buster (testing)](https://packages.debian.org/buster/kubectx) and [Sid (unstable)](https://packages.debian.org/sid/kubectx)):
+
+``` bash
+sudo apt install kubectx
```

Comment:
can you provide the commands to add one of the package repos above? it would be much better if folks can copy paste and it works (rather than trying to figure out how to add buster or sid).